packages/admin-panel/src/VizBuilderApp/components/PreviewSection.js

```diff
@@ -133,6 +137,13 @@ export const PreviewSection = () => {
   const columns = useMemo(() => getColumns(reportData), [reportData]);
   const data = useMemo(() => reportData, [reportData]);
 
+  const stringifiedData = data.map(row => {
+    let rowArrays = Object.entries(row).map(([key, value]) => {
```

Comment:
```suggestion
    const rowArrays = Object.entries(row).map(([key, value]) => {
```

Just a point on the `let` vs `const` here. We have a strong preference towards `const` over `let` where possible. It enforces immutability, which keeps code simpler and a bit more bug free, but it's also just a stylistic thing. In my IDE I get a red underline under `let` here as we have an eslint rule set up to flag these instances. Do you get the same? If not let me know and I'll help ya get eslint set up properly for your vscode 👍